src/main/java/ee/tuleva/onboarding/user/UserService.java

```diff
@@ -75,6 +78,8 @@ public User registerAsMember(Long userId) {
 
     user.setMember(newMember);
 
+    applicationEventPublisher.publishEvent(new MemberCreatedEvent(user));
```

Comment:
Maybe we emit put this event _after_ a successful save?

Reply:
Good point! Thanks